releasenotes/notes/amplitude-estimation-refactor-92dd0a2905b6fa0e.yaml

```diff
@@ -0,0 +1,12 @@
+---
```

Comment:
This reno does not seem to include the new FasterAE that was added

Reply:
Good catch, thanks